### PR TITLE
WD : Feat watch later and playlist management

### PR DESCRIPTION
**Pull request by devananth**
### Watchlater Feature:
        - user can add/remove video to/from watch later
        - if unauthorized user tries add/remove, they will be redirected to login page
        - Specific watchlater page is available

### Playlist Management Feature:
        - user can create a new playlist
        - user can add/remove video to/from a playlist
        - if unauthorized user tries add/remove, they will be redirected to login page
        - Pages are available for listing all playlists and videos of a single playlist

### File to be checked:

**Context:**
     - watchlater-context.js
     - modal-context.js
     - playlist-context.js
 
**Components:**
      - PlaylistCard
      - PlaylistModal
      - VideoCard

**Pages:**
       - WatchLater.jsx
       - Playlist.jsx
       - PlaylistDetails.jsx

**Demo Gif:**
LensPlay_playlistFeature

### CODE REVIEW

File: src/components/playlistModal/playlistModal.css [hunk 1]
```diff
@@ -0,0 +1,50 @@
+.modal__wrapper{
+    background-color: var(--overlay-color);
+    position: fixed;
+    top : 0;
+    left : 0;
+    right : 0;
+    bottom : 0;
```
Comment (mansipadgelwar) [inline src/components/playlistModal/playlistModal.css]: you can use inset property here

**Review by mansipadgelwar — APPROVED** [review]
looks like your preview link is broken. fix it. code looks clean.

File: src/components/playlistModal/playlistModal.css (hunk 1 above)
Comment (devananth) [reply src/components/playlistModal/playlistModal.css]: Thanks, didn't knew it before.

**Review by nerdyspook — APPROVED** [review]
Code looks good to me. Try fixing the link for preview.

**Pull request merged**